modules/calib/src/multiview_calibration.cpp

```diff
@@ -328,12 +328,12 @@ static void optimizeLM (std::vector<double> &param, const RobustFunction &robust
         cnt_valid_frame = 0;
         for (int i = 0; i < NUM_FRAMES; i++ ) {
             if (!valid_frames[i]) continue;
-            for (int k = 1; k < NUM_CAMERAS; k++ ) { // skip first camera as there is nothing to optimize
+            for (int k = 0; k < NUM_CAMERAS; k++ ) { // for the first camera, take caution that only the frame camera is to be optimized
```

Comment:
The comment is not clear. I propose to state: Pose for camera #0 is not optimized (coordinates center), but it's re-projection error is taken into account.